Makefile

```diff
@@ -152,15 +152,16 @@ endif
 define scorecard-setup
 @$(CLUSTER_CLIENT) get namespace $(SCORECARD_NAMESPACE) >/dev/null 2>&1 &&\
 	echo "$(SCORECARD_NAMESPACE) namespace already exists, please remove it with \"make clean-scorecard\"" >&2 && exit 1 || true
-$(CLUSTER_CLIENT) create namespace $(SCORECARD_NAMESPACE)
+$(CLUSTER_CLIENT) create namespace $(SCORECARD_NAMESPACE) && \
+	kubectl label --overwrite namespace $(SCORECARD_NAMESPACE) pod-security.kubernetes.io/warn=restricted pod-security.kubernetes.io/audit=restricted
```

Comment:
This `kubectl` should be `$(CLUSTER_CLIENT)`.

Reply:
Thanks updated!